index.html

```diff
@@ -13,7 +13,7 @@
       src="https://kit.fontawesome.com/4e96f49cc5.js"
       crossorigin="anonymous"
     ></script>
-    <title>Document</title>
+    <title>Promise Uka || Portfolio</title>
```

Comment:
- [ ] Since you will use this project as your portfolio, kindly replace `My logo` with your name.
- [ ] Please add a background image to match the Figma design
- [ ] Since it is required to stick to Figma design as much as possible, kindly use the same social media icons like the ones in the design.
Figma design:
![feedfigma](https://user-images.githubusercontent.com/53347966/176995260-058a8d13-f12c-4457-bcee-489fc0af7761.JPG)
 
Your design:
![feed4](https://user-images.githubusercontent.com/53347966/176995299-c9ce6e74-7d2a-4423-ad2d-7b76c93531e5.JPG)

Reply:
Please not that previous reviews asked I change the img tag I used for the figma icons, so I had to switch to font-awesom icons that makes us of the <i> tag